LLM based coding documentation (#279)
exasim-project/NeoFOAM: README.md

- *tests* demonstrating that NeoFOAM and OpenFOAM deliver identical results are provided by this repository in the test folder.
- *examples* provides examples of how NeoFOAM can be used for writing applications
- *tutorials* provides tutorial cases which can be run like typical OpenFOAM cases

## Agentic coding

We provide an `AGENTS.md` file to support LLM-based coding workflows.

> Please instruct tools such as Claude Code to reference this file, for example by prompting: “See `AGENTS.md` for shared project instructions.”

Reviewer: The current wording might unintentionally imply a Claude-specific workflow. It could be slightly broader.
```suggestion
Please instruct AI coding tools to reference this file, for example by prompting: “See `AGENTS.md` for shared project instructions.”
```
